static/back_nav.css

```diff
@@ -8,6 +8,7 @@
 	background-color: grey;
 	padding-top: 20px;
 	color: #fff;
+	z-index: 1000;
```

Comment:
why do we need that? z-index is a very edge case solution, can it be solved somehow differently?

Reply:
Oh yes, I had some problems with the sidebar covering up the page, was testing.